control_panel_api/helm.py

```diff
@@ -30,6 +38,16 @@ def init_user(self, username, email, fullname):
             '--set', f'Fullname={fullname}',
         )
 
+    def delete_user(self, username):
```

Comment:
Don't know if you want to make this method a bit more explicit and rename to something like `uninstall_init_user_chart()`.

Similarly the `delete_namespace()` method could be renamed to `uninstall_user_charts()`.

Not a big deal, reason why I'm mentioning this is because the user namespace is actually created as part of the installation of the `init_user` helm chart, therefore `delete_namespace` could be a bit misleading.

Reply:
I would go the other way - helm shouldn't know about "users", so these methods (`delete_user`, `config_user`, `init_user`) really belong in elsewhere, probably in the calling code. But this module is really just a collection of helper functions, so I don't think it matters much.